tool/tsh/common/reexec_unix.go

```diff
@@ -51,7 +51,7 @@ func replaceStdin() (*os.File, error) {
 	}
 	var dupErr error
 	if ctrlErr := rc.Control(func(fd uintptr) {
-		dupErr = syscall.Dup2(int(fd), syscall.Stdin)
+		dupErr = dup2(int(fd), syscall.Stdin)
 		// stdin is not O_CLOEXEC after dup2 but thankfully the three stdio
 		// file descriptors must be not O_CLOEXEC anyway, so we can avoid
 		// a linux-specific implementation or syscall.ForkLock shenanigans
```

Comment:
We're not avoiding linux-specific implementations anymore so this comment is outdated. 😢

Reply:
@espadolini I updated the comment similar to above. Please let me know if I have it right.